### DISCUSSION TITLE
AOT for .NET

### DISCUSSION THREAD

**Discussion by sekkit** [nxrighthere/UnrealCLR#104] [General]
https://github.com/dotnet/runtimelab/tree/feature/NativeAOT
Official solution

**Comment by nxrighthere** [comment]
I noticed this [back in October](https://twitter.com/nx_righthere/status/1318217012833849346) but it's not usable for platforms that we want to support, at the moment.

**Comment by nxrighthere** [comment]
Relevant https://github.com/dotnet/runtime/issues/61231
